src/BLEMidiIn.h

```diff
@@ -8,6 +8,8 @@ __________           .___      .__  .__                 _____  .__       .__
                                                                                    (c) 2018-2020 alf45star
                                                                        https://github.com/alf45tar/PedalinoMini
  */
+#ifndef BLEMIDIIN_H_
```

Comment:
I discovered recently another way to write header guard that is shorter and nicer I guess.
You can use `#pragma once` instead of `#ifndef ... #define ...` and you don't need the closing guards.

Reply:
Hi I know about pragma once but ifdef is more widely supported.
Also you can check if an header is already included in another file using the ifdef

C file
#ifdef thisheader_h
Do something
#endif